src/fare-contracts/details/DetailsContent.tsx

```diff
@@ -59,6 +61,14 @@ export const DetailsContent: React.FC<Props> = ({
   const {tariffZones, userProfiles} = useFirestoreConfiguration();
   const {deviceInspectionStatus, barcodeStatus} = useMobileTokenContextState();
 
+  // Checks if the FareContract is purchased by a different ID,
+  // then if yes, return the purchaser ID, otherwise return blank.
+  const purchasedByAccountId =
+    fc.purchasedBy !== fc.customerAccountId ? fc.purchasedBy : '';
```

Comment:
because of the !== I think it returns true o false, not blank if I am not wrong. perhaps only `fc.purchasedBy !== fc.customerAccountId` might work.